samples/google/sheets/program.py

```diff
@@ -5,52 +5,39 @@
 """
 
 import autokitteh
-from autokitteh.google import google_id, google_sheets_client
-from autokitteh.slack import slack_client
+from autokitteh.google import google_sheets_client
 
 
 sheet = google_sheets_client("sheets_conn").spreadsheets().values()
-slack = slack_client("slack_conn")
 
 
-def on_slack_slash_command(event):
+def on_http_get(event):
     """Use a Slack slash command to interact with a Google Sheet.
 
     See: https://api.slack.com/interactivity/slash-commands, and
     https://api.slack.com/interactivity/handling#message_responses
 
     In this sample, we expect the slash command's text to be either:
-    - A Google Sheets ID (https://developers.google.com/sheets/api/guides/concepts)
-    - A full Google Sheets URL (to extract the spreadsheet ID from it)
+    - A valid Google Sheets ID (https://developers.google.com/sheets/api/guides/concepts)
 
     Args:
         event: Slack event data.
     """
-    user_id = event.data.user_id
-    sheet_id = _extract_sheet_id(event.data.text, user_id)
+    params = event.data.url.query
+    sheet_id = params.get("sheet_id")
```

Comment:
No need to clarify what `event.data.url.query` is, it's clear (and short) enough.

Also, while `sheet_id` is descriptive, there's no need to be that specific in the URL when we're accepting only one type of ID.

```suggestion
    sheet_id = event.data.url.query.get("id")
```

Reply:
Done